docs/src/main/asciidoc/doc-reference.adoc

```diff
@@ -124,13 +125,6 @@ Use all lowercase letters, separate words with hyphens, and avoid symbols and sp
 Prefix:: Use a common prefix to group-related documents.
 Documents related to writing and contributing to Quarkus docs share the `doc-` prefix, for example.
 
-Suffix:: The file name should reflect the document type:
-
-- Concept documents should end in `-concept.adoc`
-- How-to guides should end in `-howto.adoc`
-- References should end in `-reference.adoc`
-- Tutorials should end in `-tutorial.adoc`
-
```

Comment:
I see you kept the suffix in the existing documents so I'm a bit confused: should the suffix be kept or not?

Reply:
I think it is optional. For micrometer, I kept it (because the suffix works to disambiguate in that case). I'll clean up the other cases.

Reply:
@ebullient hold on. My question was not coming out of nowhere: for most cases, we will need to disambiguate because we will have several contents for each component. Thus why I'm wondering about if getting rid of the suffix is a good idea.

Reply:
Yes. language update is that you can use them where it makes sense:
```
** Use the filename syntax of`<category>-<titlekeyword>-<titlekeyword>.adoc`. For example, `security-basic-authentication.adoc`.
** Include the diataxis type (concept, howto, reference, tutorial) in the file name if it makes sense to do so.
For example, `telemetry-micrometer.adoc` is a reference, and `telemetry-micrometer-tutorial.adoc` is a tutorial.
```

Reply:
My problem with that is we might end up with a lot of future renaming and redirects and I'm not sure it's going to be an improvement.

Reply:
I agree with that... though removal of the "Required" may mean less redirection than there otherwise would have been, as we can refactor more of the existing guides in-situ and preserve their current names.

Reply:
That's not really true given, in most cases, you will split the guides in several parts (howto and reference especially).

Feel free to merge this (except there is a conflict now) but my guess is that this particular change of policy won't be an improvement.

Reply:
Trying for a compromise. I hear what you're saying, and I think the file name ending will come into play when disambiguation is necessary. I've updated to this: 

```
Suffix:: Use a suffix that reflects the document type (optional):

- `-concept.adoc` for concept documents
- `-howto.adoc` for how-to guides
- `-reference.adoc` for references 
- `-tutorial.adoc` for tutorials
```

As far as the yaml generator goes, the type is still inferred from the file name if the attribute is missing. So we can find a way forward and see what works best. I know the security doc names do get really long w/ the _required_ suffix.